lib/state.js

```diff
@@ -468,10 +485,11 @@ StateResolver.prototype.resolveVariable_ = function(name, value) {
   } else if (value.isObject()) {
     data.varTableIndex = this.getVariableIndex_(value);
     var maxProps = this.config_.capture.maxProperties;
-    if (maxProps && maxProps < Object.keys(value.value()).length) {
+    var numKeys = Object.keys(value.value()).length;
+
+    if (!isEvaluated && maxProps && maxProps < numKeys) {
```

Comment:
I was looking at it from the perspective if the max number of properties was `10`, for example, and an Object has `10` properties, then we shouldn't display an error. So I don't think `<=` is needed.